flight/PiOS/STM32/pios_servo.c

```diff
@@ -391,7 +414,18 @@ int PIOS_Servo_SetMode(const uint16_t *out_rate, const int banks, const uint16_t
 				case SHOT_DSHOT300:
 				case SHOT_DSHOT600:
 				case SHOT_DSHOT1200:
+#if defined(PIOS_INCLUDE_DMASHOT)
+					if (PIOS_DMAShot_IsConfigured() && PIOS_DMAShot_RegisterServo(chan)) {
```

Comment:
I already had a partial confirmation, when I was testing on Seppuku without slaving TIM14, resulting in mixed DMA and GPIO DShot. Just tested by commenting out PIOS_DMAShot_Init() in pios_board.c on Seppuku while leaving the #define intact. Motors started spinning.

Pudding for mixed DShot sources (before I changed call order in pios_servo). Channel 2 is GPIO, rest is DMA:
![dualshot](https://cloud.githubusercontent.com/assets/4010813/25203925/80ca04f6-255b-11e7-81e4-651353756abf.png)